swap_meet/clothing.py

```diff
@@ -1,2 +1,22 @@
-class Clothing:
-    pass
\ No newline at end of file
+from .item import Item
+
+# Clothing Class extends Item Class
+class Clothing(Item):
+    def __init__(self, id=None, fabric="Unknown", condition=0.0):
+        super().__init__(category="Clothing", condition=condition, id=id)
+        self.fabric = fabric
+        self.condition = condition
+
+    
+    # return f"An object of type Clothing with id {id_value}. It is made from {fabric_value} fabric."
+    def get_category(self):
+        # return self.__class__.__name__
+        return "Clothing"
```

Comment:
Prefer this function to dynamically return the class name like you have on line 13 instead of being hardcoded to return a string "Clothing". If this class name changed from `Clothing` to `Clothes`, you would need to remember to change line 14, but if you directly return `self.__class__.__name__` then this function would be set up to accurately return the correct class name.

```suggestion
        return self.__class__.__name__
```

Reply:
We actually can remove `get_category` from `Clothing` completely since your parent class `Item` has the same method. 

Currently, `get_category` in `Item` is also hardcoded to return a string ("Item"), but if you refactor the parent class's method to dynamically return a class name then, any type of item could inherit from it and get the category name.

I'll follow up this comment in `item.py` directly too.